internal/impl/tmpl/envrc.tmpl

```diff
@@ -1,15 +1,7 @@
 # Automatically sets up your devbox environment whenever you cd into this
```

Comment:
What's the difference between `envrc.tmpl` and `envrcContent.tmpl`?

Reply:
`envrc.tmpl` is the file itself. But inside the file we use a devbox command and that command prints what is inside `envrcContent.tmpl`